composer.json

```diff
@@ -30,38 +30,38 @@
     "require": {
         "php": "^5.5.9|~7.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.7|^3.0|^4.0",
-        "symfony/debug": "^2.7|^3.0|^4.0",
-        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
-        "symfony/event-dispatcher": "^2.7|^3.0|^4.0",
-        "symfony/finder": "^2.7|^3.0|^4.0",
-        "symfony/framework-bundle": "^2.7|^3.0|^4.0",
-        "symfony/http-foundation": "^2.7|^3.0|^4.0",
-        "symfony/http-kernel": "^2.7|^3.0|^4.0",
-        "symfony/routing": "^2.7|^3.0|^4.0",
-        "symfony/security-core": "^2.7|^3.0|^4.0",
-        "symfony/templating": "^2.7|^3.0|^4.0",
+        "symfony/config": "^3.4|^4.0",
+        "symfony/debug": "^3.4|^4.0",
+        "symfony/dependency-injection": "^3.4|^4.0",
+        "symfony/event-dispatcher": "^3.4|^4.0",
+        "symfony/finder": "^3.4|^4.0",
+        "symfony/framework-bundle": "^3.4|^4.0",
+        "symfony/http-foundation": "^3.4|^4.0",
+        "symfony/http-kernel": "^3.4|^4.0",
+        "symfony/routing": "^3.4|^4.0",
+        "symfony/security-core": "^3.4|^4.0",
+        "symfony/templating": "^3.4|^4.0",
         "doctrine/inflector": "^1.0",
         "willdurand/negotiation": "^2.0",
         "willdurand/jsonp-callback-validator": "^1.0"
     },
     "require-dev": {
         "sensio/framework-extra-bundle": "^3.0.13|^4.0|^5.0",
         "symfony/phpunit-bridge": "^4.1.8",
-        "symfony/asset": "^2.7|^3.0|^4.0",
-        "symfony/form": "^2.7|^3.0|^4.0",
-        "symfony/validator": "^2.7|^3.0|^4.0",
+        "symfony/asset": "^3.4|^4.0",
+        "symfony/form": "^3.4|^4.0",
+        "symfony/validator": "^3.4|^4.0",
         "symfony/serializer": "^2.7.11|^3.0.4|^4.0",
-        "symfony/yaml": "^2.7|^3.0|^4.0",
-        "symfony/security-bundle": "^2.7|^3.0|^4.0",
-        "symfony/web-profiler-bundle": "^2.7|^3.0|^4.0",
-        "symfony/twig-bundle": "^2.7|^3.0|^4.0",
-        "symfony/browser-kit": "^2.7|^3.0|^4.0",
+        "symfony/yaml": "^3.4|^4.0",
+        "symfony/security-bundle": "^3.4|^4.0",
+        "symfony/web-profiler-bundle": "^3.4|^4.0",
+        "symfony/twig-bundle": "^3.4|^4.0",
+        "symfony/browser-kit": "^3.4|^4.0",
         "symfony/dependency-injection": "^2.7.20|^3.0|^4.0",
         "symfony/expression-language": "~2.7|^3.0|^4.0",
-        "symfony/css-selector": "^2.7|^3.0|^4.0",
+        "symfony/css-selector": "^3.4|^4.0",
         "phpoption/phpoption": "^1.1",
-        "jms/serializer-bundle": "^2.0|^3.0",
+        "jms/serializer-bundle": "^2.3.1|^3.0",
```

Comment:
needed for the `deps=low` job which would otherwise fail because of schmittjoh/JMSSerializerBundle#625